features/pensioner/pensioner_disregard.feature

```diff
@@ -35,6 +35,31 @@ Feature:
     And I should see the following "capital summary" details:
       | attribute                     | value   |
       | total_capital                 | 19800.0 |
-      | pensioner_capital_disregard  | 20000.0  |
+      | pensioner_capital_disregard   | 20000.0 |
       | assessed_capital              | 0.0     |
       | pensioner_disregard_applied   | 19800.0 |
+
+  Scenario: A applicant with a partner with capital
+    Given I am undertaking a certificated assessment with a pensioner applicant who is not passported
+    And I am using version 5 of the API
+    And I add the following employment details:
+      | client_id |     date     |  gross | benefits_in_kind  | tax   | national_insurance |
+      |     C     |  2022-07-22  | 200.50 |       0           | 75.00 |       15.0         |
+      |     C     |  2022-08-22  | 200.50 |       0           | 75.00 |       15.0         |
+      |     C     |  2022-09-22  | 200.50 |       0           | 75.00 |       15.0         |
+    And I add the following additional property details for the partner in the current assessment:
+      | value                       | 170000.00 |
+      | outstanding_mortgage        | 100000.00 |
+      | percentage_owned            | 100       |
+      | shared_with_housing_assoc   | false     |
+      | subject_matter_of_dispute   | false     |
+    When I retrieve the final assessment
+    Then I should see the following "capital summary" details:
+      | attribute                     | value   |
+      | total_capital                 | 0.0     |
+      | pensioner_capital_disregard   | 80000.0 |
+      | assessed_capital              | 0.0     |
+      | pensioner_disregard_applied   | 0.0     |
```

Comment:
I am nervous because this test doesn't touch the bug, does it? Do we not need one where some of the disregard is being applied, across both people?

Reply:
It does - the scenario is now eligible whereas before it was ineligible. I'll add some extra workings out to show that

Reply:
Ah ok so this is pensioner disregard applied to partner's capital? Which didn't happen before.
But I was expecting also a test of pensioner disregard applied to both client's capital and partner's capital. I think that would be valuable. I know with our current implementation, if one works the other works, but that's not true of all implementations.

Reply:
Also I don't actually understand how if `pensioner_disregard_applied` is 0 this has changed, but maybe that will become clear when you show the extra stuff